scaleway/helpers_lb.go

```diff
@@ -207,3 +207,29 @@ func expandLbHCHTTPS(raw interface{}) *lb.HealthCheckHTTPSConfig {
 		Code:   expandInt32Ptr(rawMap["code"]),
 	}
 }
+
+func expandLbCertificateLetsEncrypt(raw interface{}) *lb.CreateCertificateRequestLetsencryptConfig {
+	if raw == nil || len(raw.([]interface{})) != 1 {
+		return nil
+	}
+	rawMap := raw.([]interface{})[0].(map[string]interface{})
```

Comment:
nit: add a new line above.

Reply:
sorry? between the if block and rawmap or above the if?

Reply:
Yes, it's not important but just to improve visibility. :)